molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisPluginInterceptor.java

```diff
@@ -60,7 +60,9 @@ public void postHandle(HttpServletRequest request, HttpServletResponse response,
 			{
 				modelAndView.addObject(KEY_PLUGIN_ID, molgenisPlugin.getId());
 			}
-			
+
+
+			modelAndView.addObject("path", molgenisSettings.getProperty("resourcePath"));
```

Comment:
Define a static final String somewhere for resourcePath to prevent code breaking on name change.